### ISSUE TITLE
Management reporting for organisation admins

### ISSUE THREAD

**Issue by baszoetekouw** [SURFscz/SBS#1186]
As an organisation admin, I want to be able to see/download (export to csv) certain statistics about my organisation. Such as:
- How many COs
- Which services are used by those COs
- How many internal users (only COs of this organisation) use which service through SRAM
- How many external users (which are a member of a CO of this organisation) use which service through SRAM
  - How many of those come from within SURFconext (which IdPs), eduGAIN (which IdPs) and eduID

Some form of time scale management should be possible, e.g. per week/month/quarter/year.

In principle, it makes sense that the Org admin would like to have more insight in what happens in his ogf.  However, we need more information to be able to determine which information is useful to solve the usecases of the institution.  Risk is that we are implementing stuff that is not useful for the customers.

**Comment by logan-life** [comment]
[SD-62826](https://servicedesk.surf.nl/jira/browse/SD-62826)

**Comment by logan-life** [comment]
We need to understand what problem the stats are solving so that we can determine if we want to do this and with what prio.

**Comment by logan-life** [comment]
This came up in discussion with a stakeholder, will refine more.

**Comment by logan-life** [comment]
Refined more, will also check with stakeholder on further acceptance criteria. 

See [SRAM-260](https://jira.ia.surf.nl/browse/SRAM-260)

**Comment by logan-life** [comment]
I agree with past Logan. Before we expand functionality for notifying people about stuff, we first need to have our event streaming solution significantly improved.

**Comment by logan-life** [comment]
Won't do. Combination of pre-conditions w/r/t audit history AND scope widening for this sort of feat. in SXS context.
